src/polymer/polymer2-element-scanner.ts

```diff
@@ -133,6 +132,7 @@ export class Polymer2ElementScanner implements JavaScriptScanner {
     const className = element.clazz.namespacedName;
     let tagName: string|undefined = undefined;
     // TODO(rictic): support `@customElements explicit-tag-name` from jsdoc
+    // TODO(justinfagnani): do this in this PR?
```

Comment:
recommend against doing this now, as this whole thing is getting a big rewrite in #625